src/utils.ts

```diff
@@ -98,6 +99,18 @@ const getWalletPosition = (isViewportSmall: boolean): IWalletPosition => {
   return position
 }
 
+export function verifyMode(w: WalletTypes, a: AppMode | undefined): AppMode {
+  const allowedModes = ModeWalletTypeRelation[w]
+  if (a) {
```

Comment:
If app mode is `AppMode.Full`, or 0, the if condition will coerce to false.

```suggestion
  if (a !== undefined) {
```

Reply:
Updated in commit.